build-tools/packages/build-tools/src/fluidBuild/tasks/groupTask.ts

```diff
@@ -33,7 +33,7 @@ export class GroupTask extends Task {
 				if (prevTask !== undefined) {
 					const leafTasks = new Set<LeafTask>();
 					prevTask.collectLeafTasks(leafTasks);
-					task.addDependentLeafTasks(leafTasks.values());
+					task.addDependentLeafTasks(leafTasks);
```

Comment:
Is there something we could do to the function typing to prevent this sort of mistake? Maybe it's one of those things where both _could_ be correct, and this was just a case where the wrong value was used?

Reply:
I was surprised that worked before.  but from the typescript iterable definition, the SetIterator has @@iterator field, which makes it an iterable :(   Not sure what to do here.

Reply:
Having a hard time figuring out what the error is though. Passing a Set, or <Set>.values() seems like it should be the same thing in this case?